patch/0012-drivers-hwmon-support-LTC3888.patch

```diff
@@ -0,0 +1,116 @@
+From e32f1ade04454f2c3df1599c61fbd5e562ab7e05 Mon Sep 17 00:00:00 2001
+From: Ashwin H <Ashwin.H@amd.com>
+Date: Tue, 27 Jun 2023 06:32:53 +0000
+Subject: [PATCH 12/14] drivers/hwmon: Adding support LTC3888
```

Comment:
Please use imperative mood, and look through `git log drivers/hwmon/pmbus/ltc2978.c` for the prefixes:

> hwmon: (pmbus/ltc2978) Add support for LTC3888

For the commit message body see [commit 228b687d9e20 (hwmon: (ltc2978) Add support for LTC3886)](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=228b687d9e20f367e4d5ea8723e3abbf0892db61).

Reply:
> Please use imperative mood, and look through `git log drivers/hwmon/pmbus/ltc2978.c` for the prefixes:
> 
> > hwmon: (pmbus/ltc2978) Add support for LTC3888
> 
> For the commit message body see [commit 228b687d9e20 (hwmon: (ltc2978) Add support for LTC3886)](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=228b687d9e20f367e4d5ea8723e3abbf0892db61).

I will check

Reply:
Done.